bouncer/views.py

```diff
@@ -75,6 +77,12 @@ def annotation(self):
           pretty_url = pretty_url + jinja2.Markup("&hellip;")
 
         statsd.incr("views.annotation.200.annotation_found")
+        visibility = 'public'
+        if ( 'private' in self.request.GET.keys() and 
+              self.request.GET['private'] == 'true' ):
+            visibility = 'private'
+            quote = None
+            text = None
```

Comment:
Even though we need to change how bouncer discovers that an annotation is not public, we still need to have tests where given a private annotation this view will return `None` for both the quote and the text.